dvc/fs/dvc.py

```diff
@@ -253,3 +255,9 @@ def info(self, path_info):
                 ret[obj.hash_info.name] = obj.hash_info.value
 
         return ret
+
+    def _download(self, from_info, to_file, **kwargs):
```

Comment:
For the record: this is a bit ugly, but it will turn into a cleaner and more proper `fs.get_file` when migrating dvcfs to fsspec.